app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
It is a good practice to place each argument on a new line and leave opening and closing parenthesis without arguments on the same line in the `__init__`. Follow this approach in the future.